recipes/jupyterlab-classic/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "jupyterlab-classic" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "0f918c987ea88d19c2a099cf45e3c4b29ae014e85b335e7abdb2a499280fe46d" %}
```

Comment:
No need to create a variable that will be used in a single  place.
BTW, try [grayskull](https://github.com/conda-incubator/grayskull) to create your recipes. Things like these will be done automatically for you.

Reply:
Nice thanks, grayskull looks great!

Reply:
my kingdom for `grayskull go` and `grayskull rust`...